### DISCUSSION TITLE
SmoothGraphics.arc is not smooth.

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7733] [Q&A]
![image](https://user-images.githubusercontent.com/87828192/130339069-0327f645-39fa-4ee6-b709-2cf32302d086.png)

when I'm using "arc" method  in SmoothGraphics, 
It looks like stairs.
But I can't use Graphics, it looks bad than SmoothGraphics.

**Comment by bigtimebuddy** [comment]
Please report this as an issue here: https://github.com/pixijs/graphics-smooth/issues/new

**Comment by ivanpopelyshev** [comment]
Its geom issue, not antialiasing. Curves in pixi are separated to small lines, and if you work with zoom, sometimes you get this thing.
That's what being used by both Graphics and SmoothGraphics: https://github.com/pixijs/pixijs/blob/dev/packages/graphics/src/utils/ArcUtils.ts . Its using settings from GraphicsUtils, you have to change them. 

No, i wont give you exact code because I think that users have to clone pixi and look in separate IDE window each time they use particular not-obvious feature, code is the best documentation :)
